CONTRIBUTING.md

```diff
@@ -10,7 +10,6 @@
     - [Issue assignment & Communication](#issue-assignment--communication)
     - [Creating a Draft Pull Request](#creating-a-draft-pull-request)
     - [Formatting Pull Request Description](#formatting-pull-request-description)
-    - [Disabling GitHub Actions](#disabling-github-actions)
```

Comment:
Your PR keeps undoing the changes from the previous PRs. I'm wondering how you did it.

Please make sure that there's no changes other than your own changes reflected in this PR. Thanks!